### DISCUSSION TITLE
Blank field with wire:model

### DISCUSSION THREAD

**Discussion by steveijar** [livewire/livewire#5653] [1. Help] [answered]
OK I'm official tearing my hair out on this one, I can't get my form field to populate with initial data. I've tried every combination I can think of and the field stays blank.

I've stripped it down to the bare minimum code and I can't see why it won't work, can anyone point me in the right direction or even just a starting point?!

This should work right?
laravel 10.0
livewire 2.12
alpinejs

I'm also using flatpickr as a datepicker but not on this page

edit: this code version is taken from the Laravel Daily video so I know it should work!
https://www.youtube.com/watch?v=N9r_Rhumr80&ab_channel=LaravelDaily

Edit2: laravel Sail on docker, not sure that would make any difference!?

Route file
`Route::get('/liveedit', AuctionLiveEdit::class);
`

AuctionLiveEdit.php
```
<?php

namespace App\Http\Livewire;

use App\Models\House;
use Livewire\Component;

class AuctionLiveEdit extends Component
{

    public $houses = [];

    protected $rules = [
        'house.*.name' => 'required|string',
    ];

    public function render()
    {
        $this->houses = House::all()->toArray();
        return view('livewire.auction-live-edit', ['houses' => $this->houses]);
    }

}
```

livewire.auction-live-edit.blade.php

```
<div>
    <div class="grid grid-cols-1 gap-4">      


        @foreach ($houses as $index => $house)
            Name: {{ $house['name'] }}
            <input type="text" wire:model="house.{{ $index }}.name" />          
    
        @endforeach
    

    </div>
</div>
```

**Comment by joshhanley** [comment, marked as answer]
@steveijar it looks like you have a typo on your wire:model on the input field, you have `house.` instead of `houses.`.

And the same typo in the rules array. 

You also shouldn’t pass `['houses' => …]` into your view in the render method as the public property `$houses` is already made available to the view for you by Livewire.

Also, not related, but you’d be better off making the `House::all()…` call in a mount method instead of render, as putting it in render means it will run on every request making database calls. Where as putting it in mount means it will run once and then Livewire will keep track of the state for you, as you are assigning it to a public property.

Hope this helps!

**Reply by steveijar** [reply]
that was it, I can't believe I spent so long looking at it I just couldn't see it!

I get what you mean about passing the houses on render I just put that in to try and simplify as much as possible to see why it wasn't working!

Thanks @joshhanley

**Reply by joshhanley** [reply]
@steveijar no worries! 🙂 yeah it happens!
